README.md

```diff
@@ -115,7 +115,6 @@ that can be installed using a recent version of pip. **Currently pip
 install is only available on Linux**.
```

Comment:
Wheels should now be available for windows but need to make sure to call out that perf-analyzer option is not available on windows.